components/PageHeader.vue

```diff
@@ -1,47 +1,116 @@
 <template>
-  <div class="header">
-    <h2 class="pageTitle">
+  <div class="PageHeader mb-3">
+    <h2 class="PageTitle">
       <v-icon v-if="iconPath" size="4rem" class="mr-2">
         {{ iconPath }}
       </v-icon>
-      <slot />
+      {{ title }}
     </h2>
+    <div v-if="updatedAt" class="UpdatedAt">
+      <span>{{ $t('最終更新') }}</span>
+      <time :datetime="convertDate(updatedAt)">{{
+        formatDate(updatedAtAsDate)
+      }}</time>
+    </div>
+    <div v-show="!['ja', 'ja-basic'].includes($i18n.locale)" class="Annotation">
+      <span>{{ $t('注釈') }}</span>
+    </div>
   </div>
 </template>
 
 <script lang="ts">
 import Vue from 'vue'
 
-export default Vue.extend({
+import { convertDatetimeToISO8601Format } from '@/utils/formatDate'
+
+type Data = {}
+type Methods = {
+  formatDate(date: Date): string
+  convertDate(dateAsString: string): string
+}
+type Computed = {
+  updatedAtAsDate: Date
+}
+type Props = {
+  iconPath: string
+  title: string
+  updatedAt: string
+}
+
+export default Vue.extend<Data, Methods, Computed, Props>({
   props: {
     iconPath: {
       type: String,
       required: false,
       default: '',
     },
+    title: {
+      type: String,
+      required: true,
+      default: '',
+    },
+    updatedAt: {
+      type: String,
+      required: true,
+      default: '',
+    },
+  },
+  computed: {
+    updatedAtAsDate() {
+      return new Date(this.updatedAt)
+    },
+  },
+  methods: {
+    formatDate(date) {
+      return `${this.$d(date, 'dateTime')} JST`
+    },
+    convertDate(dateAsString) {
+      return convertDatetimeToISO8601Format(dateAsString)
+    },
   },
 })
 </script>
 
 <style lang="scss" scoped>
-.header {
+.PageHeader {
   display: flex;
   align-items: flex-end;
   flex-wrap: wrap;
-}
 
-.pageTitle {
-  @include font-size(30);
+  @include lessThan($small) {
+    flex-direction: column;
+    align-items: baseline;
+  }
 
-  color: $gray-2;
-  display: flex;
-  align-items: center;
-  line-height: 1.35;
-  font-weight: normal;
-  margin: 0 0.5em 0 0;
+  .PageTitle {
+    @include font-size(30);
 
-  @include lessThan($small) {
-    @include font-size(20);
+    color: $gray-2;
+    display: flex;
+    align-items: center;
+    line-height: 1.35;
+    font-weight: normal;
+
+    @include lessThan($small) {
+      @include font-size(20);
+    }
+
+    @include largerThan($small) {
+      margin-right: 2em;
```

Comment:
```suggestion
      margin-right: 0.5em;
```
以前のを引き継ぐ形で `0.5em` がよいかと思います 🙏

Reply:
@kaizumaki そのようにします

Reply:
あと、やっぱりすみません！ `.UpdatedAt` に `margin-bottom: .2rem;` はあったほうがよさそうです 🙏

Reply:
@kaizumaki `.UpdatedAt` に `margin-bottom` を付けると，`.Annotation` が非表示の場合に多少表示が崩れるのが気になるので，`.Annotation` に `margin-top` を付けました．cf. 460cece

Reply:
おお、すみません、、現在のサイトではこちら↓のように下が揃っているのですよね。これを再現したいのです。
![スクリーンショット 2021-05-23 19 16 10](https://user-images.githubusercontent.com/14883063/119256401-89eb0b80-bbfb-11eb-9762-04751aeffcb2.png)

Reply:
@kaizumaki なるほど，承知しました．
e0671d2 のようにしました．